python/paddle/trainer_config_helpers/layers.py

```diff
@@ -2819,6 +2824,12 @@ def step(input):
 
     :type targetInlink: LayerOutput|SubsequenceInput
 
+    :param is_train: recurrent_group is used for training (True) or generating (False).
+                     If is training, one of the input type must be LayerOutput; else, 
```

Comment:
The comment is inaccurate. recurrent_group may have three modes: training, testing and generating. In training and testing mode, inputs must contain LayerOutput, while in generating, must not. Therefore,  it is more accurate if the parameter "is_train" is changed to "is_generating".